cli_reference/openshift_developer_cli/odo-release-notes.adoc

```diff
@@ -5,10 +5,95 @@ include::modules/common-attributes.adoc[]
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760585[Bug 1760585] The `odo delete --all` command deletes the `$HOME/.odo` folder when run from `$HOME`.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760589[Bug 1760589] Autocompletion does not work for the `--context` flag in `odo push`.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1761442[Bug 1761442] The `component create` command fails when used with `--context` flag and `--binary` flag if the binary is located in a temporary folder.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1783179[Bug 1783179] When you set an environment variable, create an URL route and then change the source code of the component, the URL route becomes inaccessible.
+
+[id="odo-known-issues_{context}"]
+== Known issues
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760574[Bug 1760574] A deleted namespace is listed in the `odo project get` command.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760575[Bug 1760575] The `odo app delete` command removes application components but not services.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760577[Bug 1760577] The `odo push` command does not delete the OpenShift objects when the component name is changed.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760586[Bug 1760586] The `odo delete` command starts an infinite loop after a project is deleted and a component name is set.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760588[Bug 1760588] The `odo service create` command crashes when run in Cygwin.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760590[Bug 1760590] In Git BASH for Windows, the `odo login -u developer` command does not hide a typed password when requested.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1783188[Bug 1783188] In a disconnected cluster, `odo component create` command throws an error `...tag not found...` despite the component being listed in the catalog list.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1761440[Bug 1761440] It is not possible to create two services of the same type in one project.
+
+[id="odo-technology-preview_{context}"]
+== Technology Preview features in {odo-title}{nbsp}{odo-ver}
+
+[NOTE]
+====
+Technology Preview features are not fully supported, may not be functionally complete, and are not suitable for deployment in production. Customers are encouraged to provide feedback and functionality suggestions for a Technology Preview feature before it becomes fully supported.
+====
+
+* `odo debug` is a feature that allows users to attach a local debugger to a component running in the Pod on {product-title}. 
++
+Use the `odo debug port-forward` command to start the port forwarding.
++
+Use the `odo config set DebugPort 9292` command to specify the remote port on which the debugging agent should run.
```

Comment:
This sentence is confusing. Suggest: "...to specify the remote port that the debugging agent should on."